backend/fms_core/viewsets/_fetch_data.py

```diff
@@ -9,6 +9,7 @@
 from fms_core.services.library import convert_library_concentration_from_ngbyul_to_nm
 
 from ..utils import decimal_rounded_to_precision
+from ._utils import fix_count_derived_samples
```

Comment:
do not call this fix_ ... just name the variable to describe what it really is (count_derived_by_sample).